NEWS

```diff
@@ -20,6 +20,10 @@ issues]].
   and Christian Egli.
 - Add support for back translation in Russian literary braille thanks
   to Andrey Yakuboy.
+- In Russian literary braille, if the letters 'Ё' and 'Й' consist of two code
+  points, they are displayed correctly.
+- Add new Russian contracted (grade 1) table thanks to Andrey Yakuboy and
+  Ekaterina Anisimova.
```

Comment:
Should we mention that it is an experimental table?

> Ekaterina Anisimova

It would be good to mention her as author and/or copyright holder in the table and/or test file.

Reply:
I think it's excessively.

Reply:
Oh, I mean mention that this table is experimental.
Ekaterina is on the copyright in the table, I've also added her in 'Authors'.

Reply:
Yes, I saw that you added her to AUTHORS, but to justify this she should also be mentioned in the table and/or test, which is not the case.

Reply:
She is in the copyright section of the table:
```
#  Copyright (C) 2021 Kate <musicate@yandex.ru>
```
She wanted to use nickname instead of full name.

I've created tests by myself, so her name isn't in the test-file.

Reply:
I see. Hmm, not sure what to think of this. I guess as long as the email address is there, a nickname is fine. But then why put her full name in AUTHORS? Isn't that confusing? @egli What do you think?

Reply:
We thought the full name is more correct in news-releases and in the list of authors...

Reply:
Sorry this doesn't make a lot of sense to me. Yes perhaps the NEWS entry could deviate, but AUTHORS should just reflect what is in the source files.

Reply:
I am not a copyright laywer, but I would guess it would be easier to use the same name everywhere